pyedflib/tests/test_highlevel.py

```diff
@@ -292,32 +292,51 @@ def test_anonymize(self):
         self.assertEqual(new_header['technician'], 'xxxx')
 
         with self.assertRaises(AssertionError):
-            highlevel.anonymize_edf(self.personalized, 
+            highlevel.anonymize_edf(self.personalized,
                                     new_file=self.anonymized,
                                     to_remove=['patientname', 'birthdate',
                                                'admincode', 'patientcode',
                                                'technician'],
                                     new_values=['x', '', 'xx', 'xxx'],
                                     verify=True)
-            
+
+
+    def test_crop_edf(self):
+        data_dir = os.path.join(os.path.dirname(__file__), 'data')
+        edf_file = os.path.join(data_dir, 'test_generator.edf')
+        outfile = os.path.join(data_dir, 'tmp_test_generator_cropped.edf')
+        new_start = datetime(2011, 4, 4, 12, 58, 0)
+        new_stop = datetime(2011, 4, 4, 13, 0, 0)
+        highlevel.crop_edf(
+            edf_file, new_file=outfile, new_start=new_start, new_stop=new_stop)
+        highlevel.crop_edf(
+            edf_file, new_file=outfile, new_start=None, new_stop=new_stop, verbose=False)
+        highlevel.crop_edf(
+            edf_file, new_file=outfile, new_start=new_start, new_stop=None)
```

Comment:
Can you add tests here that read the files and check for the timestamp of start/stop? :)

Reply:
Done